main/src/ca/uwaterloo/flix/language/ast/Type.scala

```diff
@@ -538,6 +538,13 @@ object Type {
     mkApply(Type.Cst(TypeConstructor.SchemaExtend(name)), List(tpe, rest))
   }
 
+  /**
+    * Returns the type `Nullable[nullity, tpe0]`.
+    *
+    * NB: Note the argument order.
+    */
+  def mkNullable(tpe0: Type, nullity: Type): Type = Apply(Apply(Cst(TypeConstructor.Nullable), nullity), tpe0)
```

Comment:
What's the advantage of reversing the argument order? Seems like it might be confusing.

Reply:
Yeah, this is a bit stupid. The problem is that in `Resolver` when I see `NonNull` I need to construct the type `Nullable[t, False`, but I cannot since I don't have `t`, but by swapping the order I can make it work.

Reply:
The proper fix is likely just to carry some richer types through the Parsed, Weeded, and Named ASTs and then deal with it that way.